app.js

```diff
@@ -104,69 +104,6 @@ function tripleMatchesSpec( triple, matchSpec ) {
   return true; // no false matches found, let's send a response
 }
 
-
-function formatChangesetBody( changeSets, options ) {
```

Comment:
these two functions were extracted unchanged to their own file to make app.js a little less full. Exception: the comment in send-request.js and the retry mechanism